packages/flutter/lib/src/material/tabs.dart

```diff
@@ -534,6 +534,7 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
     this.indicatorSize,
     this.labelColor,
     this.labelStyle,
+    this.labelPadding,
```

Comment:
Unless there is a good reason to support passing null, I think this would be better as a default argument:

this.labelPadding = kTabLabelPadding

Reply:
I originally had it the way you suggested. The rest of the (rather old) TabBar API uses null to indicate "substitute the default". So for the sake of consistency...